### DISCUSSION TITLE
openweather

### DISCUSSION THREAD

**Discussion by PMax87** [flobz/psa_car_controller#311] [Q&A]
Hi, 
i have inserted my openweather API in key in config.json, but the graph consumption by temperature doesn't work, the other graph's works well.

Is there another configuration elsewhere?

Best regards.

P.

**Comment by flobz** [comment]
It should be enough. 
Did you restart?
How many trip did you make since you add the key?
Do you have error in the log?

**Reply by PMax87** [reply]
Thanks flobz for the reply,
yes i have restarted the docker container after changing the configuration, and i have made 8 trips about 100km distance.
The log is a big file, which error should i search?

Thanks.

P.

**Reply by flobz** [reply]
You shouldn't have any error error in the log file

**Comment by PMax87** [comment]
Perfect,
i have this error recursively in the log: 2022-01-08 17:42:24,757 :: ERROR :: 300 : ?, 
and if I go to the tabs I don't see the map but a blue rectangle.

![Schermata 2022-01-08 alle 18 47 11](https://user-images.githubusercontent.com/97350863/148654299-67c98cc2-40c7-472e-a4d6-67846840f7ef.png)

**Reply by flobz** [reply]
Try another browser?

**Reply by PMax87** [reply]
I try Firefox, Safari and Chrome but doesn't work, always blue rectangle.
Maybe it's because i don't have navigator and gps in my Opel?
